docs/system-admin-guide/integrations/nextcloud/oidc-sso/README.md

```diff
@@ -0,0 +1,167 @@
+
+To do this, we first create policies that will be used to control which clients are allowed to exchange tokens for a specific audience. Go to the `realm-management` client (in the default realm this client is called `master-realm`) and switch to the Tab "Authorization" and there to the "Policies" submenu. In case the "Authorization" tab is not visible, it might be necessary to make it visible by going to "Permissions" first and enabling fine-grained permissions there. Assuming you didn't create any other policies yet, your view should look similar to this:
+
+![An empty list of policies](keycloak_empty_policies.png)
+
+Click the "Create client policy" button and select "Client" as the policy type. Other types would be possible as well (such as Client scope), but this guide will explain the setup based on a client policy. Name the policy `allowed-to-exchange-openproject-token` and add at least `nextcloud` to the list of clients, while making sure that the Logic is "Positive". Afterwards repeat the same steps, creating a policy called `allowed-to-exchange-nextcloud-token`, including `openproject` in the list of clients:
+
+![One of the two exchange policies configured above](keycloak_exchange_policy.png)
+
+We will later use these policies to control who is allowed to perform a token exchange that results in a token usable for requests to the given client. For example if you have additional clients that need to obtain a token usable at OpenProject, you'd add those clients to the `allowed-to-exchange-openproject-token` client list. The names suggested here are proposals for readable policy names, but they have no practical impact.
+
+Next we will need to use the policies, so that they actually do allow the token exchange for given clients. Navigate to the `openproject` client, go to the Tab "Permissions", enable the box "Permissions enabled" and select the permission with the name `token-exchange`. You should end up on a form similar to this one:
+
+![Empty view of the permission configuration dialog](keycloak_exchange_permission.png)
+
+In the field "Policies" you can now add the `allowed-to-exchange-openproject-token` we created earlier. This means that clients matching that policy will be allowed to exchange an access token that has the `openproject`.
+
+Afterwards repeat the previous steps to add the `allowed-to-exchange-nextcloud-token` policy to the `nextcloud` client.
+
+##### Configuring OpenProject
+
+In OpenProject it is enough to configure the ID of the Nextcloud client (in our example: `nextcloud`) as the "Storage Audience". This will make sure that OpenProject will attempt to exchange a token for that audience before making requests to Nextcloud:
+
+![OpenProject is configured to manually exchange tokens for the audience called nextcloud](openproject_manual_storage_audience.png)
+
+##### Configuring Nextcloud
```

Comment:
TODO: Explain how to configure the `user_oidc` app. At least:

1. Enable accepting Bearer Tokens
2. Enable storing login tokens